client/src/containers/ProjectPeople/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { Input, Table, Button } from 'semantic-ui-react';
+
+import { RootState } from 'typings/rootState';
+import AddPeopleModal from './AddPeopleModal';
+import UserAvatar from 'components/common/UserAvatar';
+import * as actions from './logic/actions';
+import styles from './styles.module.scss';
+import searchResult from 'assets/images/search-result.svg';
+
+const ProjectPeople = () => {
+	const dispatch = useDispatch();
+	const { t } = useTranslation();
+	const { idAction } = useSelector((rootState: RootState) => rootState.projectPeople);
+
+	const {
+		project,
+		project: { users, lead },
+	} = useSelector((rootState: RootState) => rootState.project);
+	const [searchData, setSearchData] = useState('');
+
+	const usersWithoutLead = useMemo(() => users.filter((user) => user.id !== lead.id), [users, lead.id]);
+
+	const onDeleteUser = (usersId: string): void => {
+		dispatch(actions.startDeletingUser({ usersId, project }));
+	};
+
+	const onSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+		const data = event.target.value;
+		setSearchData(data);
+	};
+
+	const filteredUsers = useMemo(() => {
+		if (searchData === '') {
```

Comment:
!searchData

Reply:
With this approach, an implicit type coercion will be used, which in my opinion may not be obvious in some cases